src/skmatter/metrics/_pairwise.py

```diff
@@ -156,12 +159,12 @@ def pairwise_mahalanobis_distances(
     observations from the same distribution.
 
     Parameters:
-        x : np.ndarray of shape (n_samples_X, n_features)
-            An array where each row is a sample and each column is a feature.
-        y : np.ndarray of shape (n_samples_Y, n_features)
-            An array where each row is a sample and each column is a feature.
+        X : np.ndarray of shape (n_samples_X, n_components)
```

Comment:
just a little reminder

```suggestion
        X : numpy.ndarray of shape (n_samples_X, n_components)
```